test/testcafe/spec/ChallengeOktaVerifyResendPushView_spec.js

```diff
@@ -106,7 +106,7 @@ test
     const challengeOktaVerifyPushPageObject = await setup(t);
     await checkA11y(t);
     await challengeOktaVerifyPushPageObject.waitForErrorBox();
-    const pageTitle = challengeOktaVerifyPushPageObject.getFormTitleWithError();
+    const pageTitle = challengeOktaVerifyPushPageObject.getFormTitle();
```

Comment:
don't need to use `getFormTitleWithError()` since the form title does not share an h2 tag with the error title anymore